components/home/OurServices.vue

```diff
@@ -3,8 +3,8 @@
     <v-container>
       <v-stack direction="column" align="start">
         <app-section-descriptor
-          title="Arcana’s Privacy Stack"
-          heading="Four Decentralized Services To Secure Your Apps. Four Less Reasons To Worry"
+          title="Services"
```

Comment:
title and heading have same synonyms, can we find another name for title or heading?

Reply:
Do you have any suggestions? Any title and heading are distinct, title resembles what the entire document is about and heading resembles the title of a specific section in that document